src/oatpp/encoding/Unicode.cpp

```diff
@@ -87,7 +87,7 @@ v_int32 Unicode::encodeUtf8Char(const char* sequence, v_buff_size& length){
       length = 3;
     } else if((byte | 8) != byte){
       length = 4;
-      v_int32 value = *((p_int32)sequence);
+      v_int32 value = *(reinterpret_cast<p_int32>(const_cast<char*>(sequence)));
```

Comment:
no need to remove const - let's cast to const p_int32

Reply:
doesn't work (like in Base64.cpp)